src/test/php/load-coverage-tokens.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
```

Comment:
Would be nice to have a short explanation of what this file is for beyond "load coverage tokens".

The why would be interesting.

```suggestion
<?php
/**
 * {WHAT WE DO} {WHY}
 */
```